apps/system/js/remote/system_remote.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+(function(exports) {
+  var SystemRemote = {
```

Comment:
Since this is already in remote dir, let's rename to app.js to be sync with local system